Sage/carbon
# feat(label): describe label using design tokens

### Proposed behaviour

Describes the label and icon-wrapper using design tokens. Removes all 'theme' in css files. 
Removes unused props and adds missing ones in label.style.js file.
Changes the label color for the disabled state.
Updates tests after the above changes. 

### Current behaviour

The labels use the theme styled-component property.
The current label color for the disabled state is rgba(0,0,0, 0.55).
The propTypes in the css files are missing used props and left unused props.


### Checklist




#### QA


### Additional context



### Testing instructions
Code sandbox: https://codesandbox.io/s/carbon-quickstart-forked-3blrr

The correct appearance of the labels can be checked in the following components: `textarea`, `switch`, `radio-button`, `inline-inputs`, `checkbox`. 
For theme: `mint` and `aegean` there should be no differences.
Use devtools to check if labels use **css variables**.
